src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,92 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
+    private static final int columnsNumber = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        final int linesNumber = countLines(fromFileName);
+
+        String[][] operationData = new String[linesNumber][columnsNumber];
+        operationData = readFromFile(fromFileName);
```

Comment:
You don't need to read from this file twice. No need to "count lines"
```suggestion
        String[][] operationData = readFromFile(fromFileName);
```